src/main/java/com/internship/socialnetwork/dto/PostDto.java

```diff
@@ -0,0 +1,44 @@
+package com.internship.socialnetwork.dto;
+
+import com.internship.socialnetwork.model.Post;
+import com.internship.socialnetwork.model.User;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import jakarta.validation.constraints.NotNull;
+
+@Data
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
```

Comment:
Please try to sort annotations in some order for example `@Data, @Builder, @NoArgs, @AllArgs`